client/my-sites/stats/activity-log-day/index.jsx

```diff
@@ -34,25 +34,17 @@ const DAY_IN_MILLISECONDS = 1000 * 60 * 60 * 24;
  * classifier and the event itself (for rendering)
  *
  * @param {Array} logs sorted activity log items
- * @param {Number} activityId selected rewind operation
+ * @param {?Number} rewindId selected rewind operation
  * @returns {Array<String, ?Object>} pairs of [ classifier, event ]
  */
-const classifyEvents = ( logs, activityId ) => {
-	/** @type {Array<Array<Object, ?Object>>} contains pairs of [ log, next log ] **/
-	const logPairs = zip( logs, logs.slice( 1 ) );
-
-	return flatMap( logPairs, ( [ log, nextLog ] ) => {
-		if ( log.activityId === activityId ) {
-			return [ [ 'rewind-confirm-dialog', {} ], [ 'event', log ] ];
-		}
-
-		if ( nextLog && nextLog.activityId === activityId ) {
-			return [ [ 'event-before-dialog', log ] ];
-		}
-
-		return [ [ 'event', log ] ];
-	} );
-};
+const classifyEvents = ( logs, { rewindId = null } ) =>
```

Comment:
why use an object as the second param instead of the plain `rewindId`?

Reply:
also, does that work with jsdoc here? the param is really an object and you need to doc the rewindId prop?

Reply:
We’re about to add another item, the `backupId` - these are visible in the linked PR and commits

Reply:
Not sure on JSDoc - I’ve seen this pattern before and PhpStorm is happy with it. I don’t mind changing if need be